doc/manifest/schema/1.0.0/installer.md

```diff
@@ -0,0 +1,444 @@
+# Windows Package Manager
+## Manifest Schema v1.0.0 "installer" File
```

Comment:
nit: Manifest Schema v1.0.0 Installer File to match the pattern in above default locale doc?

Reply:
the other file would be something like "locale.en-US" so I didn't quote the other one, but when the title matches the name of the file I quoted it. I could remove quotes...